pycbc/events/eventmgr.py

```diff
@@ -608,21 +608,28 @@ def __init__(self, opt, ifos, column, column_types, network_column,
         for ifo in self.ifos:
             self.event_index[ifo] = 0
         self.event_index['network'] = 0
-        self.template_event_dict['network'] = \
-                                numpy.array([], dtype=self.network_event_dtype)
+        self.template_event_dict['network'] = numpy.array(
+            [], dtype=self.network_event_dtype)
 
-    def cluster_template_network_events(self, tcolumn, column, window_size):
+    def cluster_template_network_events(self, tcolumn, column, window_size,
+                                        slide=0):
         """ Cluster the internal events over the named column
         """
-        cvec = self.template_event_dict['network'][column]
-        tvec = self.template_event_dict['network'][tcolumn]
-        if window_size == 0:
-            indices = numpy.arange(len(tvec))
-        else:
+        if not window_size == 0:
```

Comment:
This function could definitely do with some comments ... We don't want to rewrite the code and end up with something as opaque as coh_PTF is :-)